CMakeLists.txt

```diff
@@ -145,7 +147,15 @@ endif()
 # 3rd-party
 #
 
-if (NOT TARGET ggml)
+if (LLAMA_USE_SYSTEM_GGML)
+    message(STATUS "Using system-provided libggml, skipping ggml build")
+    find_package(ggml REQUIRED)
+    set(GGML_TARGET ggml::ggml)
+else()
+    set(GGML_TARGET ggml)
```

Comment:
instead of using a `GGML_TARGET` var, we should just add the `ggml::ggml` alias target

Reply:
That indeed simplifies the approach.